src/server/server.js

```diff
@@ -1,22 +1,28 @@
 import {SocketCluster} from 'socketcluster';
-import {auth0Secret} from './secrets';
 import path from 'path';
-// import os from 'os';
+import { getDotenv } from '../universal/utils/dotenv';
+
+// Import .env and expand variables:
+getDotenv();
+
+const clientSecret = process.env.AUTH0_CLIENT_SECRET ||
```

Comment:
this isn't a client secret, it's a key that only the server should have. If an attacker got this, he could write his own JWTs...